### PR TITLE
Adding creation date in the clip.

### PR DESCRIPTION
**Pull request by krman009**
It'll be displayed in the bottom-right corner of the clip.

Feel free to ping me in case you've any doubts. 😉

### CODE REVIEW

File: src/components/Local.vue [hunk 1]
```diff
@@ -98,12 +99,19 @@ export default {
   },
 
   methods: {
+    getNow() {
+      const today = new Date();
+      const month = today.toLocaleString("default", { month: "long" });
+      const date = today.getDate() + " " + month + ", " + today.getFullYear();
+      return date + " " + today.getHours() + ":" + today.getMinutes();
+    },
```
Comment (sereneinserenade) [inline src/components/Local.vue:107]: thank you for showing me how to use the date in JS  :-) , I always found it intemedating !!

File: src/components/Local.vue [hunk 2]
```diff
@@ -149,4 +157,16 @@ export default {
...
+  right: 0.36em;
+  bottom: 0.18em;
+  font-size: 0.85em;
+}
+/* better look for the small devices */
+@media (max-width: 720px) {
+  .small-info {
+    font-size: 0.65em;
+  }
+}
```
Comment (sereneinserenade) [inline src/components/Local.vue:171]: ah you really are a CSS king man !!

File: src/components/Local.vue [hunk 3]
```diff
@@ -61,6 +61,7 @@
           </v-btn>
         </span>
         <span>{{ clip.title }}</span>
+        <span class="small-info"> Created on: {{ clip.created_on }} </span>
```
Comment (sereneinserenade) [inline src/components/Local.vue:64]: should we add this feature, where the date field is a link and onclick, we show all the clips generated on that date ?

File: src/components/Local.vue (hunk 1 above)
Comment (krman009) [reply src/components/Local.vue:107]: Actually I also googled for this and I mixed two answers from the Stackoverflow to achieve my desired date format. :smile:

File: src/components/Local.vue (hunk 2 above)
Comment (krman009) [reply src/components/Local.vue:171]: Nope, it's kinda basic CSS :smiley:

File: src/components/Local.vue (hunk 3 above)
Comment (krman009) [reply src/components/Local.vue:64]: "Filtering by Date" is a good thing IMO. What if we implement it in the filter panel (which is yet to be developed). Let me know what you think.

Comment (sereneinserenade) [reply src/components/Local.vue:64]: I had never given thought to filter panel, I'd add this to a TODO in project, Also

I think that there is also discussions panel in Github now, so I would try to use that if possible from now on.

File: src/components/Local.vue (hunk 1 above)
Comment (sereneinserenade) [reply src/components/Local.vue:107]: I know those stackoverflow searches right !!

File: src/components/Local.vue (hunk 2 above)
Comment (sereneinserenade) [reply src/components/Local.vue:171]: I think I should stop using frameworks excessively and use my own CSS 😖

**Pull request merged**